src/migration/tasks/AddVisualIDPropertyToTagsTask.ts

```diff
@@ -20,23 +20,24 @@ export default class AddVisualIDPropertyToTagsTask extends MigrationTask {
   }
 
   async migrateTenant(tenant: Tenant): Promise<void> {
-    let tags = await TagStorage.getTags(tenant.id, {}, { limit: Constants.BATCH_PAGE_SIZE, onlyRecordCount: true, skip: 0 });
-    const count = tags.count;
-    let skip = 0;
+    let tags: Tag[];
     let updated = 0;
     do {
-      tags = await TagStorage.getTags(tenant.id, {}, { limit: Constants.BATCH_PAGE_SIZE, skip: skip, sort:{ _id:1 } });
-      if (!Utils.isEmptyArray(tags.result)) {
+      // Get the tags
+      tags = await global.database.getCollection<any>(tenant.id, 'tags')
+        .find({ visualID: { $exists: false } })
+        .limit(Constants.BATCH_PAGE_SIZE)
+        .toArray();
+      if (!Utils.isEmptyArray(tags)) {
         const visualID = new ObjectID().toHexString();
-        for (const tag of tags.result) {
+        for (const tag of tags) {
           await global.database.getCollection<any>(tenant.id, 'tags').update(
```

Comment:
```suggestion
          await global.database.getCollection<any>(tenant.id, 'tags').updateOne(
```
Silence a deprecation warning.